docs/csv.md

```diff
@@ -0,0 +1,115 @@
+# CSV import
+
+`reccmp` has the option to parse [comma-separated value (CSV)](https://en.wikipedia.org/wiki/Comma-separated_values) files and load data to supplement code annotations on the original binary.
```

Comment:
I like the "how" part of this README! What I'm still missing is a bit of the "why", e.g.:
- everything works fine without CSV, so this feature is optional
- you may have additional data from other sources, and export into CSV is easier than into source code
- annotating the size of objects is not supported in code annotations
- ... (I have probably forgotten about several aspects)